### ISSUE TITLE
Issue regarding running  Asgard 1.5.1 on EC2 (ubuntu)  

### ISSUE THREAD

**Issue by sainivikas** [Netflix/asgard#722]
I followed guidelines of Quick start and face following error when running Asgard on my Ubuntu EC2 machine.

AWS Error Code: AccessDeniedException, AWS Error Message: User: arn:aws:iam::A/C no:user/ABC is not authorized to perform: swf:ListDomains on resource: arn:aws:swf:us-east-1:A/C no:/domain/*

This is my Config.groovy ->

grails {
	awsAccounts=['A/C']
	awsAccountNames=['A/C':'prod']
}
secret {
	accessId='accessid'
	secretKey='secretkey'
}
cloud {
	accountName='prod'
	publicResourceAccounts=['amazon']
}

Am i missing something ?

**Comment by arbabnazar** [comment]
@sainivikas are you able to solve this problem?

**Comment by sainivikas** [comment]
Yes, it was occur because the aws key which I used did not have appropriate permission.
